packages/components/src/DataList/components/DataListItemActions/DataListItemActions.css

```diff
@@ -1,13 +1,12 @@
 .menu {
   display: flex;
   position: absolute;
-  top: calc(var(--space-small) * -1);
+  top: -26px;
   right: 0;
   z-index: var(--elevation-menu);
   box-shadow: var(--shadow-base);
-  padding: var(--space-smaller);
   border: var(--border-base) solid var(--color-border);
   border-radius: var(--radius-base);
   background-color: var(--color-surface);
-  gap: var(--space-smaller);
+  gap: var(--space-smallest);
```

Comment:
Keeps a pinch of space between the actions so the focus states don't encroach too heavily on sibling buttons